node/service/Cargo.toml

```diff
@@ -30,6 +30,7 @@ tokio = { version = "1.12.0", features = ["macros", "sync"] }
 cli-opt = { package = "moonbeam-cli-opt", path = "../cli-opt" }
 
 # Runtimes
+runtime-common = { path = "../../runtime/common", optional = true }
```

Comment:
Should we re-export the `runtime-common::migrations` from each runtime so that we don't take this new dependency here?